angular.json

```diff
@@ -54,6 +54,7 @@
               "namedChunks": false,
               "extractLicenses": true,
               "vendorChunk": true,
+              "commonChunk": false,
```

Comment:
Why are you disabling the common bundle?

Reply:
Speed up loading time? Only load what is really needed etc.?

Reply:
it's not disabled, 'technically'. This is just preventing that Angular inserts a default configuration. The webpack.custom still pushes artifacts into the common bundle and it is being created.

Reply:
True that. I was wondering why `ng build --stats-json` still spits out common.js.